src/taipy/gui/data/content_accessor.py

```diff
@@ -58,6 +58,15 @@ def __get_mime_from_file(self, path: pathlib.Path):
                 warnings.warn(f"({path}) cannot read mime type.\n{e}")
         return None
 
+    def __get_display_name(self, var_name: str) -> str:
+        if not isinstance(var_name, str):
+            return var_name
+        if var_name.startswith("_tpC_"):
+            var_name = var_name[5:]
+        if var_name.startswith("tpec_"):
```

Comment:
that would be an 'else', right?

Reply:
no, the var can be encapsulated in a TaipyContent holder

Reply:
Oh whao. I actually couldn't figure out where the tpC part came from. Kind of searched... then abandoned.
Makes sense. Thx